railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
@fxn it's true if the operation you want to parallelize is a cpu-bound operation. Gem installation is mostly bound by I/O.